ingest/ledger_change_reader.go

```
@@ -122,6 +124,14 @@ func (r *LedgerChangeReader) Read() (Change, error) {
 			r.pending = append(r.pending, metaChanges...)
 		}
 		return r.Read()
+	case evictionChangesState:
+		// Get contract ledgerEntry evictions
+		changes := GetChangesFromLedgerEntryEvictions(
+			r.ledgerCloseMeta.EvictedLedgerKeys(),
```

could you add an error case to `EvictedLedgerKeys` instead of ignoring the error coming back from `SetContractCode`/`SetContractData` ?